pkg/descheduler/evictions/evictions.go

```diff
@@ -266,6 +281,11 @@ func IsMirrorPod(pod *v1.Pod) bool {
 	return utils.IsMirrorPod(pod)
 }
 
+// IsStaticPod checks whether the pod is a static pod.
+func IsStaticPod(pod *v1.Pod) bool {
+	return utils.IsStaticPod(pod)
```

Comment:
I know in this PR it's just following the pattern that's already set, but is there any reason we have these single-line wrappers in this package? It seems like we could get rid of them all (`IsMirrorPod`, `IsStaticPod`, `IsPriorityPod`, etc) and just call directly to `utils.IsXXX()`